ngraph/core/reference/include/ngraph/runtime/reference/relu.hpp

```diff
@@ -33,15 +33,6 @@ namespace ngraph
                     out[i] = arg[i] > zero ? arg[i] : zero;
                 }
             }
-            template <typename T>
```

Comment:
Please double check if this function was left for reference purposes. Otherwise I am fine removing it as there is no usage of it